### ISSUE TITLE
Reproducing results in the paper

### ISSUE THREAD

**Issue by z-fabian** [DPS2022/diffusion-posterior-sampling#4]
Hi, I am trying to reproduce the results from the paper and I cannot find exactly which 1k images of the FFHQ and ImageNet dataset were used for the tables in the paper. Can you please clarify the exact split used for comparing DPS with the other methods?
Thank you!

**Comment by z-fabian** [comment]
The score network for FFHQ (taken from [this](https://github.com/jychoi118/ilvr_adm) repository according to the paper) has been trained on the full FFHQ dataset with all 70k images. If the experiments in the paper have been performed on a subset of the FFHQ dataset using the above model, it seems to me that the model has been trained on the images it is being evaluated on and therefore the results might be misleading. Thanks in advance for clarifying this!

**Comment by DPS2022** [comment]
Hello @z-fabian,

Actually, we trained the score network on the subset of the FFHQ dataset with 50k images where the validation set is excluded. Specifically, we used the first 1k images of the FFHQ dataset as the validation set.

It was written incorrectly in the paper by mistake, and it is planned to be modified in the camera-ready version. We are sorry to confuse you and thank you for pointing out the important part!

**Comment by z-fabian** [comment]
Great, thank you for the clarification. I assume the folder '00000' has been used as validation set and '01000' - '49000' for training (using the default folder names after downloading the dataset), is that correct?

**Comment by DPS2022** [comment]
Exactly :)

**Comment by z-fabian** [comment]
Hi,

Could you please also publish which part of the ImageNet validation set was used as ImageNet-1k validation set for the results in the paper? Has center cropping been applied to the images after loading? I would like to reproduce the results and the plots from the paper. Thanks a lot in advance!

**Comment by sojinleeme** [comment]
Hello, 
I have a question about FFHQ dataset with 50K image folders.

Validation: '00000'
Training: '01000'-'49000' (49K)

Doesn't it include folder '50000' to be 50K images?

**Comment by OriLifschitz** [comment]
> Hi,
> 
> Could you please also publish which part of the ImageNet validation set was used as ImageNet-1k validation set for the results in the paper? Has center cropping been applied to the images after loading? I would like to reproduce the results and the plots from the paper. Thanks a lot in advance!

Any thoughts regarding this please? I too would like to know please.
